### PR TITLE
check styles fixes

### PR DESCRIPTION
**Pull request by gauravhmcts**
### Change description ###

check style fixes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java [hunk 1]
```diff
@@ -48,14 +48,30 @@ void testReportHeaderTotalsAreZeroIfNoCasesExist() {
...
+                        adhocReportType.getManuallyCreatedTotalCasesPercent()), .00);
+        assertEquals(0.00, Strings.isNullOrEmpty(
+                adhocReportType.getEt1OnlineTotalCasesPercent()) ? 0.00 : Float.parseFloat(
+                        adhocReportType.getEt1OnlineTotalCasesPercent()),.00);
+        assertEquals(0.00, Strings.isNullOrEmpty(
+                adhocReportType.getMigratedTotalCasesPercent()) ? 0.00 : Float.parseFloat(
+                        adhocReportType.getMigratedTotalCasesPercent()),.00);
+        assertEquals(0.00, Strings.isNullOrEmpty(
+                adhocReportType.getEccTotalCasesPercent()) ? 0.00 : Float.parseFloat(
+                        adhocReportType.getEccTotalCasesPercent()),.00);
```
Comment (Itay-Kirsh) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java:75]: Delete extra line

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BulkHelperTest.java [hunk 2]
```diff
@@ -247,130 +255,147 @@ public void getJurCodesCollectionWithHide_ListFromString() {
...
+                new ArrayList<>(), BulkHelper.getJurCodesListFromString(" ")));
     }
 
     @Test
     public void getJurCodesCollectionWithHide_AllJurCodesAreFiltered_ReturnsEmptyString() {
-        JurCodesTypeItem JurCodesTypeItem1 = getJurCodesWithOutcome("A", "Acas conciliated settlement");
-        JurCodesTypeItem JurCodesTypeItem2 = getJurCodesWithOutcome("B", "Withdrawn or private settlement");
-        JurCodesTypeItem JurCodesTypeItem3 = getJurCodesWithOutcome("C", "Input in error");
-        List<JurCodesTypeItem> jurCodesTypeItems = new ArrayList<>(Arrays.asList(JurCodesTypeItem1, JurCodesTypeItem2, JurCodesTypeItem3));
+        JurCodesTypeItem JurCodesTypeItem1 = getJurCodesWithOutcome(
```
Comment (Itay-Kirsh) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BulkHelperTest.java:268]: Change variable name to not start with capital

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkCreationServiceTest.java [hunk 3]
```diff
@@ -343,90 +374,125 @@ void updateBulkRequestPendingStateException() throws IOException {
...
     }
 
     @Test
      void bulkCreationLogic() {
-        BulkRequestPayload bulkRequestPayload = bulkCreationService.bulkCreationLogic(getBulkDetails(YES, "Single"),
+        BulkRequestPayload bulkRequestPayload = bulkCreationService.bulkCreationLogic(
+                getBulkDetails(YES, "Single"),
                 bulkCasesPayload, "authToken", BULK_CREATION_STEP);
         assertThat(bulkRequestPayload.getBulkDetails().toString())
         .hasToString(RESULT);
```
Comment (Itay-Kirsh) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkCreationServiceTest.java:416]: Incorrect indentation on hasToString

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkCreationServiceTest.java [hunk 4]
```diff
@@ -445,33 +512,41 @@ void retrievalCasesForPreAcceptRequestWithEmptyCaseIds() throws IOException {
...
         assertEquals("[]", casesForPreAcceptRequest.toString());
     }
 
     @Test
      void retrievalCasesForPreAcceptRequestException() throws IOException {
-        when(ccdClient.retrieveCasesElasticSearch(anyString(), anyString(), any())).thenThrow(new InternalException(ERROR_MESSAGE));
+        when(ccdClient.retrieveCasesElasticSearch(anyString(), anyString(), any()))
+                .thenThrow(new InternalException(ERROR_MESSAGE));
         assertThrows(CaseCreationException.class,
                 ()->{
```
Comment (Itay-Kirsh) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkCreationServiceTest.java:525]: whitespaces around the arrow

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerServiceTest.java [hunk 5]
```diff
@@ -345,12 +346,19 @@ public void struckOutRespondentFirstToLast() {
...
+        assertEquals("Antonio Vazquez", caseData
+                .getRespondentCollection().get(0).getValue().getRespondentName());
+        assertEquals(NO, caseData.getRespondentCollection()
+                .get(0).getValue().getResponseStruckOut());
+        assertEquals("Roberto Dondini", caseData
+                .getRespondentCollection()
+                .get(1).getValue().getRespondentName());
+        assertEquals(NO, caseData.getRespondentCollection()
+                .get(1).getValue().getResponseStruckOut());
+        assertEquals("Juan Garcia", caseData.getRespondentCollection().
```
Comment (Itay-Kirsh) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerServiceTest.java:358]: . should be on the next line

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SubMultipleServiceTest.java [hunk 6]
```diff
@@ -130,7 +137,8 @@ public void populateFilterDefaultedAllDynamicListLogic() {
         String result = "DynamicFixedListType(value=DynamicValueType(code=999999, label=Select All), " +
```
Comment (Itay-Kirsh) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SubMultipleServiceTest.java:137]: A few checkstyle issues missed on this file. Not just line longer than 120 characters but also '+' on new line and distance between varaibles.

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

**Review by Itay-Kirsh — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BulkHelperTest.java (hunk 2 above)
Comment (gauravhmcts) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BulkHelperTest.java]: done

**Review by jthmcts — APPROVED** [review]

**Review by nickwest1 — CHANGES_REQUESTED** [review]
Apart from the fact I prefer the Hamcrest approach to asserts, I appreciate you're cleaning up existing code so not part of this change.
Otherwise I've added some feedback in that is worth dealing with this time round :)

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/BulkActionsControllerTest.java [hunk 7]
```diff
@@ -196,7 +202,8 @@ public void createSearchBulkCase() throws Exception {
 
     @Test
     public void midSearchBulkCase() throws Exception {
-        when(bulkSearchService.bulkMidSearchLogic(isA(BulkDetails.class), isA(Boolean.class))).thenReturn(bulkRequestPayload);
+        when(bulkSearchService.bulkMidSearchLogic(
+                isA(BulkDetails.class), isA(Boolean.class))).thenReturn(bulkRequestPayload);
```
[the remark this answers is not in the document]
Comment (gauravhmcts) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/BulkActionsControllerTest.java:206]: Fixed it here, in whole project there are 1010 occurrences, we need a separate ticket for this

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/UpdateDataModelBuilderTest.java [hunk 8]
```diff
@@ -25,14 +25,14 @@
...
-    final String RESPONDENT_REPRESENTATIVE_NAME = "Lionel Hutz";
-    final String UNKNOWN_VALUE = "Unknown";
+    private final String CLAIMANT_REPRESENTATIVE_NAME = "Ruth Powers";
+    private final String JURISDICTION = "DSO";
+    private final String RESPONDENT_NAME = "Mindy Simmons";
+    private final String JUDGEMENT_ID = "1234-5678-9012";
+    private final String JUDGEMENT_TYPE = "Case Management";
+    private final String RESPONDENT_REPRESENTATIVE_ID = "5555-6666-7777-8888";
+    private final String RESPONDENT_REPRESENTATIVE_NAME = "Lionel Hutz";
+    private final String UNKNOWN_VALUE = "Unknown";
```
[the remark this answers is not in the document]
Comment (gauravhmcts) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/UpdateDataModelBuilderTest.java]: made them static

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReportTest.java [hunk 9]
```diff
@@ -85,9 +85,11 @@ void shouldNotShowSubmittedCase() {
         // When I request report data
         // Then the case should not be in the report data
 
-        submitEvents.add(caseDataBuilder.withManagingOffice(TribunalOffice.NEWCASTLE.getOfficeName()).buildAsSubmitEvent(SUBMITTED_STATE));
+        submitEvents.add(caseDataBuilder.withManagingOffice(
+                TribunalOffice.NEWCASTLE.getOfficeName()).buildAsSubmitEvent(SUBMITTED_STATE));
 
-        var reportData = hearingsToJudgmentsReport.runReport(ENGLANDWALES_LISTING_CASE_TYPE_ID, TribunalOffice.NEWCASTLE.getOfficeName());
+        var reportData = hearingsToJudgmentsReport.runReport(
+                ENGLANDWALES_LISTING_CASE_TYPE_ID, TribunalOffice.NEWCASTLE.getOfficeName());
```
[the remark this answers is not in the document]
Comment (gauravhmcts) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReportTest.java]: yeah done as much as I can, I think there are over 3600 vars being used. But I'll clean them up as I see them while fixing remaining check style issues :)

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkCreationServiceTest.java [hunk 10]
```diff
@@ -247,85 +252,110 @@ void caseCreationRequestWithEmptyCaseIds() throws IOException {
...
 
     @Test
      void caseCreationRequestExceptionElasticSearch() throws IOException {
         when(ccdClient.retrieveCasesElasticSearchForCreation(anyString(), anyString(), anyList(), anyString()))
                 .thenThrow(new InternalException(ERROR_MESSAGE));
-        assertThrows(CaseCreationException.class,
-                ()-> bulkSearchService.bulkCasesRetrievalRequestElasticSearch(getBulkDetails(YES, "Single"), "authToken", true, true));
+        assertThrows(CaseCreationException.class, () ->
+                bulkSearchService.bulkCasesRetrievalRequestElasticSearch(getBulkDetails(
+                        YES, "Single"), "authToken", true, true));
```
[the remark this answers is not in the document]
Comment (gauravhmcts) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkCreationServiceTest.java:267]: have put () -> on new line

**Review by nickwest1 — APPROVED** [review]

**Pull request merged**